submodules/user/user.js

```diff
@@ -531,13 +533,25 @@ define(function(require) {
 					'caller_id.internal.name': { regex: self.i18n.active().callflows.user.validation.caller_id.name },
 					'caller_id.external.name': { regex: self.i18n.active().callflows.user.validation.caller_id.name },
 					'caller_id.emergency.name': { regex: self.i18n.active().callflows.user.validation.caller_id.name },
+					'caller_id.asserted.name': { regex: self.i18n.active().callflows.user.validation.caller_id.name },
 					'caller_id.internal.number': { regex: self.i18n.active().callflows.user.validation.caller_id.number },
 					'caller_id.external.number': { regex: self.i18n.active().callflows.user.validation.caller_id.number },
-					'caller_id.emergency.number': { regex: self.i18n.active().callflows.user.validation.caller_id.number }
+					'caller_id.emergency.number': { regex: self.i18n.active().callflows.user.validation.caller_id.number },
+					'caller_id.asserted.number': { regex: self.i18n.active().callflows.user.validation.caller_id.number },
+					'caller_id.asserted.realm': { regex: self.i18n.active().callflows.user.validation.caller_id.realm }
 				}
 			});
 
-			timezone.populateDropdown($('#timezone', user_html), data.data.timezone || 'inherit', {inherit: self.i18n.active().defaultTimezone});
+			timezone.populateDropdown($('#timezone', user_html),
```

Comment:
Either keep everything on a single line or break each parameter on their own but don't mix both (mostly for clarity):
```suggestion
			timezone.populateDropdown(
				$('#timezone', user_html),
				_.get(data.data, 'timezone', 'inherit'),
				{
					inherit: self.i18n.active().defaultTimezone
				}
			);
```

Also, when breaking down your parameters that way, your closing parenthesis should also be on its own line.